lib/vdsm/storage/exception.py

```diff
@@ -115,6 +115,22 @@ def __init__(self, volume, *args):
             self.value += list(args)
 
 
+class VolumeLVMCommandError(StorageException):
+    code = 4001
+    msg = "Volume exception with LVM error"
```

Comment:
This error is not meant to be used directly, only used as a base class, so it does
not need code and msg - the inheriting class should add them.

Reply:
We can fix find_exceptions() to skip private classes (another reason to use _prefix).

Reply:
> Or maybe make the base class not inherit from anything and use multiple inheritance in the child classes.

Multiple inheritance when you have to `__init__` multiple super class does not sound promising. Lets try to keep it simple.